app/controllers/Controller.scala

```diff
@@ -98,23 +98,23 @@ abstract class LoggedInController(mcc: MessagesControllerComponents) extends Bas
 
 abstract class ApplicationController(mcc: MessagesControllerComponents) extends LoggedInController(mcc) with ActionBuilders {
   val applicationService: ApplicationService
-  val fraudPreventionConfigProvider: FraudPreventionConfigProvider
+  val fraudPreventionConfig: config.FraudPreventionConfig
```

Comment:
No need for the `config` prefix here if you import it properly at the top of the file.